src/development/ui/widgets/text.md

```diff
@@ -1,5 +1,8 @@
 ---
 title: Text widgets
 short-title: Text
+description: >
+  A catalog of Flutter's widgets for displaying and styling text.
```

Comment:
I've never seen using the arrow for breaking lines in yaml text. But, in this case, it doesn't seem necessary.